### PR TITLE
Trash page

### PR DESCRIPTION
**Pull request by partha8**
## Files to check
- [src/context/TrashProvider.js](https://github.com/partha8/melonn/compare/trash-page?expand=1#diff-77e133a78e132053268d975074caa5d0ff82ad47b5710f266af9287c19329c12)
- [src/hooks/useGetTrashedNotes.js](https://github.com/partha8/melonn/compare/trash-page?expand=1#diff-1369cd55c8f1d2f81972abab05c27bc7b869d5d4a681d248b038a3c3bd087198)
- [src/pages/Trash/Trash.js](https://github.com/partha8/melonn/compare/trash-page?expand=1#diff-f70fa2a614128df06e7498d76e2828249f6c943829f346f7b95c6a1291545199)

## Preview
trash gif

### CODE REVIEW

File: src/context/TrashProvider.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+      trashed: true,
+      updatedAt: note.updatedAt,
+      trashedAt: serverTimestamp(),
+      id: note.id,
+    });
+    deleteDoc(docRef);
+  };
+
+  const restoreNote = (note) => {
+    console.log(note);
```
Comment (hsnice16) [inline src/context/TrashProvider.js:37]: Is this console statement required? If not then I think, you can remove this.

**Review by hsnice16 — APPROVED** [review]
Great work, man. The code is neat, liked the way you are maintaining the spaces in the files, it makes the code more readable.
Good Job.

File: src/components/Card/Card.js [hunk 2]
```diff
@@ -8,22 +8,34 @@ import {
...
+    pinned,
+    trashed,
+    tags,
+    color,
+  } = note;
+
+  const { deleteNote, restoreNote, deleteFromTrash } = useTrashContext();
+
+  const location = useLocation();
+  console.log(location.pathname);
```
Comment (Hamza-777) [inline src/components/Card/Card.js:33]: Remove the unused console logs.

File: src/components/Card/card.module.css [hunk 3]
```diff
@@ -36,9 +36,14 @@
...
+    justify-content: center;
+    align-items: center;
     gap: 8px;
 }
 
 .utils>* {
     cursor: pointer;
+    display: flex;
+    justify-content: center;
+    align-items: center;
```
Comment (Hamza-777) [inline src/components/Card/card.module.css:48]: You can make a class for flex separately as you are using it in multiple places, will reduce the css% on github to some extent.

**Review by Hamza-777 — APPROVED** [review]
Everything looks good, nice work.

File: src/components/Card/Card.js (hunk 2 above)
Comment (partha8) [reply src/components/Card/Card.js:33]: Yup, in my future PRs I have removed them.

File: src/components/Card/card.module.css (hunk 3 above)
Comment (partha8) [reply src/components/Card/card.module.css:48]: Will do it.

File: src/context/TrashProvider.js (hunk 1 above)
Comment (partha8) [reply src/context/TrashProvider.js:37]: Removing it.

**Pull request merged**